canvasapi/todo.py

```diff
@@ -0,0 +1,6 @@
+from canvasapi.canvas_object import CanvasObject
+
+
+class Todo(CanvasObject):
+    def __str__(self):
+        return "Todo List ({})".format(self.id)
```

Comment:
`id` isn't a valid attribute for a todo item object. It looks like `type` is the closest thing to a meaningful attribute that is readily available, unless we try for digging deeper for `quiz` or `assignment`.

This `__str__` method is also lacking testing.

I'll do a quick patch